include/z64save.h

```diff
@@ -4,6 +4,28 @@
 #include "ultra64.h"
 #include "z64math.h"
 
+typedef enum {
+    /*  0 */ HUD_VISIBILITY_IDLE,
```

Comment:
from a user's perspective (looking at this without having looked at much of the code yet), was a bit confused what "idle" could mean in terms of hud visibility.

to clear this up my ideas are `_MAINTAIN`, as in maintain the current setting, but not totally sure. 
if not, a comment just stating briefly what it does would go a long way

Reply:
think this confusion mainly stems from this seeming mostly like an enum of various visibility settings, and idle is more of a state. thats really an issue with the implementation tho and not the name. so a comment is prob best

Reply:
Could be something like `HUD_VISIBILITY_NO_CHANGE`?

Reply:
`_NOOP` ezclappers (not serious ... unless)